docs/inputs.md

```diff
@@ -407,8 +407,10 @@ An enhanced browser `<select>` element, allowing the user to select one (or more
 
 * **multiple** - allow multiple options to be selected. data will be an object with options as keys, similar to checkbox-group
 * **search** - allow users to type stuff in to filter options. Extremely useful for longer options lists
-* **list** - The key `list`  is where the value is the name of a list that Amphora knows about accessible via `/<site>/_lists/<listName>`.
+* **list** - The key `list` is where the value is the name of a list that Amphora knows about accessible via `/<site>/_lists/<listName>`.
 * **options** - an array of strings or objects (with `name`, `value`, and optionally `sites`)
+* **keys** passthrough option for Keen to specify keys for input objects, especially for use when you don't control the input shape, e.g. lists. Defaults to `{label: 'name', value: 'value'}`
+* **storeRawData** normally only the `value` of each option is stored, but with this option you can store the entire input object
```

Comment:
Any thoughts on the name of this option? I'm not super proud of `storeRawData`. Also, any thoughts on the approach in general? I feel like leveraging the `keys` field in Keen is the way to go because it relies on concepts familiar to the core technologies and is general enough to fit future additions, but I'm open to other ideas..